Doc/library/asyncio-task.rst

```diff
@@ -414,6 +414,52 @@ reported by :meth:`asyncio.Task.cancelling`.
    Improved handling of simultaneous internal and external cancellations
    and correct preservation of cancellation counts.
 
+Group Cancellation
```

Comment:
I wonder if we should just not use the word "cancel" at all in this section. Name the exception `StopTaskGroup`, use "stop" or "terminate" or "exit" in the docstrings, and rename this section "Terminating a Task Group".

Because what we do really isn't similar to cancellation -- it doesn't raise a `CancelledError` (except internally to cancel individual tasks) and as a consequence doesn't have the precise semantics of cancelation in asyncio.

Reply:
I think that suggestion makes sense @gvanrossum. I agree with "Terminating a Task Group"

Reply:
+1